File: Problem35.py
import math

def primeCheck(n):
	if(n==1):
		return False
	for i in range(2, int(math.sqrt(n))+1):
		if(n%i==0):
			return False
	return True

count = 13

for i in range(101, 1000000, 2):
	istr = str(i)
	if(primeCheck(i)):
		flag = True
		for j in range(1, len(istr)):
			istr = istr[1:] + istr[0]
			if(primeCheck(int(istr))==False):
				flag = False
				break
		if(flag):
			count += 1
			print i

print "Final count is: ",
print count

File: Problem35.js
const circularPrimes = (n) => {
  const primeCheck = (num) => {
    if (num === 1) {
      return false;
    }
    for (let i = 2; i <= Math.floor(Math.sqrt(num)); i++) {
      if (num % i === 0) {
        return false;
      }
    }
    return true;
  };
  let count = 1;
  for (let i = 1; i < n; i += 2) {
    if (primeCheck(i)) {
      let flag = true;
      let circularNum = i.toString();
      for (let j = 1; j < i.toString().length; j++) {
        circularNum = circularNum.substring(1) + circularNum.substring(0, 1);
        if (primeCheck(Number(circularNum)) === false) {
          flag = false;
          break;
        }
      }
      if (flag) {
        count++;
      }
    }
  }
  return count;
};